comps/third_parties/vllm/src/build_docker_vllm.sh

```diff
@@ -21,27 +21,31 @@ cd $SCRIPT_DIR
 
 # Set default values
```

Comment:
This script is not used for release image build or CI test image build, only referred in readme.